### PR TITLE
QPID AMQP JMS Samples

### PR DESCRIPTION
**Pull request by chughts**
Add QPID AMQP JMS samples for standard Java and Quarkus/GraalVM execution. 

The samples share most of the code using symbolic links, but have their own specific, `pom.xml` main class, configuration, build and run instructions.

### CODE REVIEW

File: amqp-qpid/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+maven `pom.xml`, build and run instructions.
+
+## Run configurations and options
+Each set of samples has its own mechanism for configuration.
+  * [qpid-standard](/amqp-qpid/qpid-standard/README.md) uses command line arguments, JNDI and an associated `src/main/resources/jndi.prpperties` file.
+  * [qpid-quarkus]((/amqp-qpid/qpid-quarkus/README.md)) uses
+  a quarkus `src/main/resources/application.properties` file
+
+### Mode Options
+The mode options are the same for both set of applications. The difference is how
```
Comment (oljarast) [inline amqp-qpid/README.md:44]:
```suggestion
The mode options are the same for both sets of applications. The difference is in how
```

File: amqp-qpid/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Specify in `applications.properties` as `amqp-mqtest.appargs`
+
+#### Modes
+  * one of
+    * put (default)
+    * get
+    * browse
+      * this will run the sample as a put/pub, get/sub or browse. The put will put a set of TextMessage, BytesMessage, StreamMessage,
+      ObjectMessage and MapMessage.
+  * a integer number n
```
Comment (oljarast) [inline amqp-qpid/README.md:60]:
```suggestion
  * an integer number n
```

File: amqp-qpid/qpid-standard/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* `topic.myTopicLookup`
+  * topic to use for message publish / subscribe
+
+* `queue.myReplyQueueLookup`
+  * reply queue to use when running put / get in request / response mode
+    * if omitted reply defaults to using temporary queues.
+
+## Building the sample application jar
+A maven `pom.xml` is provided allowing you to use maven to download dependancies and build the sample. You can build the samples by running the command.
```
Comment (oljarast) [inline amqp-qpid/qpid-standard/README.md:34]:
```suggestion
A maven `pom.xml` is provided allowing you to use maven to download dependencies and build the sample. You can build the samples by running the command.
```

File: amqp-qpid/qpid-standard/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+````
+mvn clean package
+````
+
+### Running sample.
```
Comment (oljarast) [inline amqp-qpid/qpid-standard/README.md:40]:
```suggestion
### Running the sample
```

**Review by oljarast — COMMENTED** [review]
Some very small changes.

File: amqp-qpid/README.md (hunk 1 above)
Comment (chughts) [reply amqp-qpid/README.md]: done

**Review by oljarast — APPROVED** [review]

**Pull request merged**